integration-cli/docker_cli_build_test.go

```diff
@@ -3577,8 +3577,8 @@ RUN [ "$(id -u):$(id -g)/$(id -un):$(id -gn)/$(id -G):$(id -Gn)" = '1001:1001/do
 
 # Switch back to root and double check that worked exactly as we might expect it to
 USER root
+# Add a "supplementary" group for our dockerio user
```

Comment:
It is needed. I thought I left a comment for that in this pr... . It is the same fix as in https://github.com/docker/docker/pull/24725/files#diff-64d1c3a97fd8b1abf170680883e81aaaR3580 . Doesn't matter how we handle empty lines, this Dockerfile should not be parseable.